static/allowlist.json

```diff
@@ -28,17 +28,15 @@
     "https://rainbow-me.firebaseapp.com",
     "https://icons.duckduckgo.com",
     "https://*.live.ledger.com",
-    "https://rpc.zora.co",
     "https://falling-wandering-film.base-mainnet.quiknode.pro",
     "https://goerli.optimism.io",
     "https://data-seed-prebsc-1-s1.binance.org:8545",
     "https://matic-mumbai.chainstacklabs.com",
     "https://goerli-rollup.arbitrum.io/rpc",
     "https://goerli.base.org",
+    "https://rpc.zora.co",
     "https://testnet.rpc.zora.energy",
-    "https://eth-sepolia.g.alchemy.com/v2",
-    "https://eth-goerli.g.alchemy.com/v2",
```

Comment:
jw why removing the `eth-goerli`?

Reply:
@BrodyHughes Should be handled by the `*.g.alchemy.com` wildcards I believe